### PR TITLE
Error details to mailgun exceptions

### PR DESCRIPTION
**Pull request by snapshotpl**

### CODE REVIEW

File: src/SlmMail/Service/Exception/ValidationErrorException.php [hunk 1]
```diff
@@ -43,6 +43,7 @@
 
 use InvalidArgumentException;
 use SlmMail\Exception\ExceptionInterface;
+use Throwable;
```
Comment (roelvanduijnhoven) [inline src/SlmMail/Service/Exception/ValidationErrorException.php:46]: :scissors:

**Comment by roelvanduijnhoven** [comment]
Little detail, after that looks good to me :).

**Comment by snapshotpl** [comment]
Now should be ok :)

**Comment by roelvanduijnhoven** [comment]
Thanks @snapshotpl . Released a new version :).

**Pull request merged**